### DISCUSSION TITLE
why stats card 's  total Commits is wrong ??

### DISCUSSION THREAD

**Discussion by huanlirui** [anuraghazra/github-readme-stats#1895] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/40491203/181440681-32a02051-8c0a-4b10-a775-37c426dc53e8.png)
![image](https://user-images.githubusercontent.com/40491203/181440840-caef4ffd-9eb2-4603-bdd8-af6c960c9af8.png)

**Comment by rickstaa** [comment, marked as answer]
@huanlirui Thanks for your question. I think your confusing contributions with commits. See [1770#number-of-commits-is-incorrect](https://github.com/anuraghazra/github-readme-stats/discussions/1770#number-of-commits-is-incorrect) and #1679.

**Reply by rickstaa** [reply]
@huanlirui Did that answer your question?
